idea_town/frontend/static-src/app/views/header-view.js

```diff
@@ -30,20 +29,15 @@ export default BaseView.extend({
   //       to pick up csrftoken cookie changes.
   logout(evt) {
     evt.preventDefault();
-    xhr({
+    fetch(app.getDocHeadLink('x-logout', 'href'), {
```

Comment:
I'm not totally convinced we need to abstract out the account-related urls, but I'm interested to see where this abstraction takes us, so let's roll with it.

Random questions: Exactly which kinds of data do you want to specify in the DOM? Just enough to get the page connected to the API server? Is one `x-app-startup` tag, containing a serialized object, cleaner than a bunch of individual tags & running to the DOM over and over?

Something to think about for the future: it seems weird that we get the logout URI via `app.getDocHeadLink`, but the login URI (which contains a redirect endpoint, too) is just hard-coded [in the view](https://github.com/lmorchard/idea-town/blob/63-user-api-endpoint/idea_town/frontend/static-src/app/views/landing-page.js#L12). Also seems like we might want to put the csrftoken (`app.me.csrfToken`) in the same place inside the app as the endpoints?